server/backend/tests/db_helpers.py

```
@@ -13,8 +13,133 @@
+    # --- Knowledge (legacy ``Store`` surface) ---
+
+    async def count(self) -> int:
+        return await self.knowledge.count()
+
+    async def counts_by_tier(self) -> dict[str, int]:
+        return await self.knowledge.counts_by_tier()
+
+    async def domain_counts(self) -> dict[str, int]:
+        return await self.knowledge.domain_counts()
+
+    async def get(self, unit_id: str) -> KnowledgeUnit | None:
```

Ruff (0.15.15)

[error] Undefined name `KnowledgeUnit`

(F821)